packages/enzyme/src/Debug.js

```diff
@@ -32,7 +32,11 @@ function propString(prop) {
     case 'boolean':
       return `{${prop}}`;
     case 'object':
-      return '{{...}}';
+      if (prop instanceof String) {
```

Comment:
`instanceof` is never reliable, and should not be used.

If we want to detect boxed strings, we need to use the `is-string` library.